src/portable/raspberrypi/rp2040/rp2040_usb.h

```diff
@@ -67,10 +67,10 @@ struct hw_endpoint
 
     // Current transfer information
     bool active;
-    uint total_len;
-    uint len;
+    uint16_t total_len;
```

Comment:
total_len can actually be > 64KB in case of MSC, I intended to change the dcd_edpt_xfer() total_bytes from `uint16_t to uint32_t` however, the work is a bit overwhelming to change all the dcd for now.